### DISCUSSION TITLE
I get on error with model EleutherAI/gpt-neo-125M

### DISCUSSION THREAD

**Discussion by zikapanam** [geekjr/quickai#62] [Q&A]
Hi,

I've explore a little set of model, some work out of the box and other complains that they cannot be loaded.

As this model is set in the quickai code, i suppose this is not expected.
```
Traceback (most recent call last):
  File "/var/www/zap/docker/bots-discord/ai-py/test-ai.py", line 12, in <module>
    answer = q_and_a(prompt, question, model='EleutherAI/gpt-neo-125M')
  File "/home/zap/.local/lib/python3.10/site-packages/quickai/text_inferance.py", line 45, in q_and_a
    nlp = pipeline("question-answering", model=model)
  File "/home/zap/.local/lib/python3.10/site-packages/transformers/pipelines/__init__.py", line 788, in pipeline
    framework, model = infer_framework_load_model(
  File "/home/zap/.local/lib/python3.10/site-packages/transformers/pipelines/base.py", line 279, in infer_framework_load_model
    raise ValueError(f"Could not load model {model} with any of the following classes: {class_tuple}.")
ValueError: Could not load model EleutherAI/gpt-neo-125M with any of the following classes: (<class 'transformers.models.auto.modeling_tf_auto.TFAutoModelForQuestionAnswering'>,).
```

Thanks for your help,
BR,
Laurent

**Comment by geekjr** [comment]
GPT NEO has its own function. 

The setup would be like this for Q/A:
```python
from quickai import gpt_neo

prompt = """
The bus is a square, red, and has no roof. 
What color is the bus?
"""

generated_text = gpt_neo(prompt, "125M")
print(generated_text)
```

It think it should also be able to do French better cause it's a better model. Also, if you want better results, you can change 125M to the larger parameter models. Options are

350M
1.3B
2.7B

Remember that as you increase the model size, the processing time goes up, but so does the accuracy. The best choice depends on your task. 

If you want even more parameters, try the gpt_j function(if your system can run this resources wise, you will need nvidia GPU)

The code is the same, but there is no model parameter as GPTJ is only 6B.
